internal/commands/vaultsecrets/apps/create.go

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package apps
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/go-hclog"
+
+	secretsvcpreview "github.com/hashicorp/hcp-sdk-go/clients/cloud-vault-secrets/preview/2023-11-28/client/secret_service"
+	secretsvcstable "github.com/hashicorp/hcp-sdk-go/clients/cloud-vault-secrets/stable/2023-06-13/client/secret_service"
+	"github.com/hashicorp/hcp/internal/pkg/cmd"
+	"github.com/hashicorp/hcp/internal/pkg/flagvalue"
+	"github.com/hashicorp/hcp/internal/pkg/format"
+	"github.com/hashicorp/hcp/internal/pkg/heredoc"
+	"github.com/hashicorp/hcp/internal/pkg/iostreams"
+	"github.com/hashicorp/hcp/internal/pkg/profile"
+)
+
+type CreateOpts struct {
+	Ctx     context.Context
+	Profile *profile.Profile
+	Output  *format.Outputter
+	IO      iostreams.IOStreams
+	Logger  hclog.Logger
+
+	AppName       string
+	Description   string
+	StableClient  secretsvcstable.ClientService
+	PreviewClient secretsvcpreview.ClientService
+}
+
+func NewCmdCreate(ctx *cmd.Context, runF func(*CreateOpts) error) *cmd.Command {
+	opts := &CreateOpts{
+		Ctx:           ctx.ShutdownCtx,
+		Profile:       ctx.Profile,
+		Output:        ctx.Output,
+		IO:            ctx.IO,
+		StableClient:  secretsvcstable.New(ctx.HCP, nil),
+		PreviewClient: secretsvcpreview.New(ctx.HCP, nil),
+	}
+
+	cmd := &cmd.Command{
+		Name:      "create",
+		ShortHelp: "Create a new HCP Vault Secrets application.",
+		LongHelp: heredoc.New(ctx.IO).Must(`
+		The {{ template "mdCodeOrBold" "hcp vault-secrets apps create" }} command creates a new HCP Vault Secrets application.
+
+		Once an application is created, secrets lifecycle can be managed using the
+		{{ template "mdCodeOrBold" "hcp vault-secrets secret" }} command group.
+		`),
+		Examples: []cmd.Example{
+			{
+				Preamble: `Create a new application under HCP Vault Secrets:`,
```

Comment:
```suggestion
				Preamble: `Create a new application:`,
```

I think that much is implied by the command --help is being run under